app/assets/stylesheets/config/_colors.scss

```diff
@@ -0,0 +1,132 @@
+////////////////////////////////////////////////////////////////////////////////////
+// Colors. /////////////////////////////////////////////////////////////////////////
```

Comment:
This file is the only standalone file other than `import.scss` in config `folder`.  We don't want to load it in the same way as we load stuff in `import.scss` since this will actually inject CSS, and since we're importing `import` multiple times, these variables would also be imported multiple time - we don't want that.